themes/src/main/resources/theme/base/account/index.ftl

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Ideally a `lang` attribute is added as well. Not sure if we already have a theme property for that.
```suggestion
<html lang="en">
```